src/assets/contributors-map.geojson

```diff
@@ -4,15 +4,15 @@
     {
       "type": "Feature",
       "properties": {
-        "firstName": "Yun",
```

Comment:
@baderk I think you need to add a new entry for your info, not updating the existing one (Yun)

Reply:
Oh I see yes I will change that

Reply:
Fixed the change here
3e08a7a94d65e11fd55c39bfc0b2bea91ef6079d